src/json/members/aagamd.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Aagam Dalal",
+  "pennkey": "aagamd",
+  "major": "Computer Engineering",
+  "school": "Engineering",
+  "bio": "Hi! I'm Aagam. I'm studying Computer Engineering, with an interest in pursuing robotics. In my free time, I enjoy watching movies and running.",
+  "hometown": "Columbus, Ohio",
+  "team": "Penn Courses",
+  "roles": ["Backend Engineer"],
+  "photo": "https://imgur.com/a/Cks5ZRn",
```

Comment:
Yo I think this should be a direct imgur link (e.g. `https://i.imgur.com/xdxd.jpg`)